### PR TITLE
ISSUE-15 & ISSUE-28 - MOTD fixes

### LINKED ISSUE #15: Escape HTML tags from username
There's a way to exploit the Motd by using HTML codes on the players nickname.

### LINKED ISSUE #28: MOTD doesn't show background and weapon images

### PR DESCRIPTION
**Pull request by GuskiS**
closes #15 
closes #28

### CODE REVIEW

**Comment by Maxi605** [comment]
On ISSUE-28, First time checked, server crashed on analizing the body, Second time, the body is analized but you can't see the motd, doesn't deploy, Using camoflage on the dead body, server crashes (Sometimes),

Error Logs:

L 08/05/2018 - 17:57:35: [AMXX] Displaying debug trace (plugin "ttt_show_infos.amxx")
L 08/05/2018 - 17:57:35: [AMXX] Run time error 5: memory access 
L 08/05/2018 - 17:57:35: [AMXX]    [0] ttt_show_infos.sma::html_head (line 347)
L 08/05/2018 - 17:57:35: [AMXX]    [1] ttt_show_infos.sma::show_motd_info (line 287)
L 08/05/2018 - 17:57:35: [AMXX]    [2] ttt_show_infos.sma::ttt_showinfo (line 105)

**Comment by Maxi605** [comment]
Oh, and you can't use key 'B' to buy, you have to do /buy

**Comment by Maxi605** [comment]
ISSUE-15 Works fine that i've seen

File: scripting/ttt_show_infos.sma [hunk 1]
```diff
@@ -21,23 +15,10 @@ new const g_szColors[][] =
...
 };
 
-public plugin_precache()
-{
-  static icon[32];
-  for(new i = 0; i <= charsmax(g_szIconNames); i++)
-  {
-    if(i < 5 && strlen(g_szIconNames[i]) < 3) continue;
-    formatex(icon, charsmax(icon), "gfx/ttt/%s.gif", g_szIconNames[i]);
-    precache_generic(icon);
```
Comment (GuskiS) [inline scripting/ttt_show_infos.sma:31]: I think CS can't precache `.gif` images. If I use `.tga` format, I can't show it in MOTD, cause TGA is not supported by HTML 😢

**Comment by Maxi605** [comment]
Everything works fine now!

**Pull request merged**